components/LandingPage.tsx

```diff
@@ -2,97 +2,216 @@ import * as React from 'react'
+              One of the most important skills in a challenging workplace is
+              knowing how to ask for help. Our curriculum is intentionally
+              difficult and we have dedicated mentors to answer your questions
+              and guide you how to communicate effectively.
+            </p>
+          </div>
+          <div className="col-md-4">
+            <img
+              src="choose-03.svg"
+              alt="Curriculum Image"
+              style={{ height: '170px' }}
+            />
+            <h4 className="mt-3 font-weight-bold">Work Experience</h4>
+            <p className="mt-4 font-weight-light">
+              Once you have completed our curriculum and learned to communicate
+              effectively, you will be invited to contribute to this open-source
+              project to improve the learning experience for future students.
+            </p>
+          </div>
         </div>
       </div>
-      <div className="row mt-5">
-        <div className="col-md-10 offset-md-1">
-          <h4 className="font-weight-light" style={{ lineHeight: 1.5 }}>
+      <div
+        className="container-fluid py-5"
+        style={{ background: 'rgb(84,64,216)', color: 'white' }}
+      >
+        <div className="container mt-5">
+          <h1
+            id="journey"
+            className="mt-5 display-4 font-weight-bold text-center"
+          >
+            Your Learning Journey
+          </h1>
+          <h4
+            className="mt-5 font-weight-light col-md-8 offset-md-2 pb-5 text-center"
+            style={{ lineHeight: 1.5 }}
+          >
             Our learning process is interactive and follows the same practice as
             a well functioning engineering team. From the first line of code you
             write, you will be code reviewed by engineers to ensure you are
-            always following industry best practices.
+            always following industry best practice
           </h4>
+          <div className="row mt-5 pb-5">
+            <div className="col-md-5 offset-md-2 order-md-2 text-center">
+              <img src="journey-01.svg" className="mb-3 img-fluid"></img>
+            </div>
+            <div className="col-md-5 order-md-1">
+              <h2 className="font-weight-bold">Creating an account</h2>
+              <p className="mt-3 font-weight-light" style={{ lineHeight: 1.5 }}>
+                After creating an account, you will gain access to our code
+                reviewed curriculum and our community chat. You will be able to
+                setup c0d3.com CLI which will allow you to submit code for
+                review. The CLI allows you to code in Javascript from any code
+                editor of your choice.
+              </p>
+            </div>
+          </div>
+          <div className="row mt-5 pb-5">
+            <div className="col-md-5 text-center">
+              <img src="journey-02.svg" className="mb-3 img-fluid" />
```

Comment:
missing an alt tag here

Reply:
if you don't want to have an any alt, then use an empty alt attribute like this `alt=""` so that screen readers don't read out "journey-02.svg"